server/src/internalClusterTest/java/org/opensearch/indices/replication/SegmentReplicationRelocationIT.java

```diff
@@ -98,28 +105,29 @@ public void testPrimaryRelocation() throws Exception {
 
         logger.info("--> get the state, verify shard 1 primary moved from node1 to node2");
         ClusterState state = client().admin().cluster().prepareState().execute().actionGet().getState();
-
-        logger.info("--> state {}", state);
-
         assertEquals(
             state.getRoutingNodes().node(state.nodes().resolveNode(newPrimary).getId()).iterator().next().state(),
             ShardRoutingState.STARTED
         );
 
-        final int finalDocCount = initialDocCount;
-        ingestDocs(finalDocCount);
```

Comment:
I believe this was added to SegmentReplicationIT for this test case, if its no longer used lets pls remove it.